packages/strapi-plugin-upload/admin/src/containers/HomePage/tests/reducer.test.js

```diff
@@ -320,4 +320,50 @@ describe('Upload | containers | HomePage | reducer', () => {
 
     expect(reducer(state, action)).toEqual(expectedState);
   });
+
+  it('should update searchParams with given params', () => {
```

Comment:
Can you wrap your test in a `describe` with the name of the corresponding action so it is easier to identify what your are testing?